### ISSUE TITLE
Add support for table schemas in get_query_tables()

### ISSUE THREAD

**Issue by dargueta** [macbre/sql-metadata#17]
The library currently doesn't support PostgreSQL-style [table schemas](https://www.postgresql.org/docs/current/static/ddl-schemas.html). This is particularly useful when doing queries involving two tables with the same name but in different schemas, as the current behavior makes it impossible to tell the difference between the two.

Minimal example:

```py
# Current behavior:
>>> sm.get_query_tables("SELECT * FROM s.t")
['t']

# Expected:
>>> sm.get_query_tables("SELECT * FROM s.t")
['s.t']
```

Something a bit less contrived:

```sql
SELECT
  a.*
FROM product_a.users AS a
JOIN product_b.users AS b
  ON a.ip_address = b.ip_address
```

`get_query_tables()` only returns `users` here, instead of the ideal `product_a.users` and `product_b.users`.

**Comment by macbre** [comment]
@dargueta, I've published v1.3 on [PyPI](https://pypi.org/project/sql_metadata/)

**Comment by dargueta** [comment]
Thanks a lot!
